src/content/support/p1-process/index.md

```diff
@@ -0,0 +1,92 @@
+---
+linkTitle: P1 Incident Process
+title: Critical incident process
+description: Critical Incident Process description followed by the Giant Swarm support team when a priority one incident is called.
+weight: 100
+menu:
+  main:
+    parent: overview
+    weight: 20
+user_questions:
+  - What process does Giant Swarm follow in case of critical incidents?
+owner:
+  - https://github.com/orgs/giantswarm/teams/teddyfriends
+last_review_date: 2023-05-02
+---
+
+After years of handling critical enterprise workloads in production, we have hardened our incident process based on our learnings. In this document we focus on critical incidents, called `Priority 1` (P1) incidents, though some of the steps could be part of regular incidents too. Giant Swarm classify incidents in a simple manner, critical incidents when a production system is impaired or regular incidents which have a straightforward process.
```

Comment:
I think we need to review the voice through most of this doc - if it's meant to be for "internal" usage, it feels strange to say stuff like "Giant Swarm classify incidents" -> it (and other areas throughout) should rad more like "We classify incidents..."

Reply:
Well as it will be public I wanted to write it in formal way